packages/graph-engine/src/nodes/color/wheel.ts

```diff
@@ -0,0 +1,71 @@
+import { NodeDefinition, NodeTypes } from "#/types.js";
+import chroma from "chroma-js";
+
+export const type = NodeTypes.WHEEL;
+
+export type State = {
+  hueAmount: number;
+  hueAngle: number;
+  saturation: number;
+  lightness: number;
+  colors: number;
+};
+
+export const defaults: State = {
+  hueAmount: 360,
+  hueAngle: 180,
+  saturation: 80,
+  lightness: 50,
+  colors: 8,
+};
+
+type colorValue = {
+  index: number;
+  value: string;
+}
+
+
+export type Input = {
+  hueAmount: number,
+  hueAngle: number,
+  saturation: number,
+  lightness: number,
+  colors: number,
+} & State;
+
+export const process = (input: Input, state: State): colorValue[] => {
```

Comment:
We probably need to add a validateInput  here to ensure that final.colors is not negative and we end up in an infinite loop